backend/src/test/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceAsyncTest.java

```diff
@@ -343,7 +386,7 @@ void personSaveAllFailure_triggersErrorEmail() throws IOException {
   }
 
   @Test
-  @SliceTestConfiguration.WithSimpleReportStandardUser
+  @SliceTestConfiguration.WithSimpleReportOtherOrgUser
```

Comment:
⚠️ This test kept failing with the addition of the new test above. The StandardUser in this configuration had the same email as the StandardAllFacilitiesUser so something weird was happening here. Just decided to make a new user in a different org to test this scenario.